http_check/tests/test_unit.py

```diff
@@ -32,3 +32,52 @@ def test_instances_do_not_share_data():
 
     assert http_check_1.HTTP_CONFIG_REMAPPER['ca_certs']['default'] == 'foo'
     assert http_check_2.HTTP_CONFIG_REMAPPER['ca_certs']['default'] == 'bar'
+
+
+def test_message_lenght_when_content_is_too_long():
+    max_lenght = http_check.MESSAGE_LENGTH
+    try:
+        http_check.MESSAGE_LENGTH = 25
+        too_long_content = 'this message is too long'
+        error_message = 'There has been an error.'
+        message = HTTPCheck._include_content(True, error_message, too_long_content)
+    finally:
+        http_check.MESSAGE_LENGTH = max_lenght
+
+    assert len(message) == 25
+    assert error_message in message
+    assert too_long_content not in message
+
+
+def test_message_lenght_when_content_is_ok():
+    content = '''{
+    "HikariPool-1.pool.ConnectivityCheck" : {
+        "healthy" : true
+    },
+    "database" : {
+        "healthy" : true,
+        "message" : "Service located at jdbc ostgresql://pgbouncer-server.staging.net is alive. Version: 1.5"
+    },
+    "deadlocks" : {
+        "healthy" : true
+    }
+    "gateway" : {
+        "healthy" : true,
+        "message" : "Service located at https://apis.staging.eu.people-doc.com is alive."
+    }
+}'''
+    error_message = 'There has been an error.'
+    message = HTTPCheck._include_content(True, error_message, content)
+
+    assert len(message) < http_check.MESSAGE_LENGTH
+    assert content in message
+    assert error_message in message
```

Comment:
(nit) I would favour asserting the whole message instead of the 3 assertions, a bit more straightforward and readable.